services/order.py

```diff
@@ -0,0 +1,32 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import Ticket, Order
+
+from django.db import transaction
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime
+) -> None:
+    order = Order.objects.create(username=username)
+    if date:
+        order.created_at = date
+    order.save()
```

Comment:
Don't Repeat Yourself: You are calling the `save()` method twice. It's better to call it once after all changes are made.